ansys-grantami-serverapi-openapi/src/ansys/grantami/serverapi_openapi/models/granta_server_api_aggregations_aggregation_datum.py

```diff
@@ -61,7 +61,7 @@ class GrantaServerApiAggregationsAggregationDatum(ModelBase):
         "floatFunctionalGraph".lower(): "#/components/schemas/GrantaServerApiAggregationsFloatFunctionalAggregation",
     }
 
-    discriminator = "datumType"
+    discriminator = "datum_type"
```

Comment:
Do we know where this change came from? It doesnt look like a change in the swagger json?

Reply:
I'm investigating. I have made changes to the template last week because of failing tests.